# # custom error
# a = int(input("enter the value b/w 1 to 5 :" ))

# if (a>=5 or a>=1):
#     raise ValueError("value should be b/w 1 and 5")

# if we use value error then if there will be any error then code will pause here and will not be executed 


hey = (input("write 'quit' here :"))
if(hey == "quit"):
    print("u did this")
else:
    raise ValueError("this can't be exepted")


# did this : D 